Update IE scroll properties

Elchi3 opened the pull request:
(Hrm, the diffs look useless)

Originally, the IE specific scroll properties were added in https://github.com/mdn/browser-compat-data/pull/2424. In that PR, they've been added with a -ms- prefix and it was stated that support was available since IE 5 with an unknown removal version . I don't think that is correct.

First, I think they haven't been removed. 

Second, I think the -ms- prefix came much later. See https://msdn.microsoft.com/de-de/library/ms531153(v=vs.85).aspx "Windows Internet Explorer 8. The -ms-scrollbar-3dlight-color attribute is an extension to CSS, and can be used as a **synonym** for scrollbar-3dlight-color in IE8 Standards mode. "

If you look at ancient css doc sites like http://www.css4you.de/ieproperty.html, http://www.css4you.de/scrollbar-3dlight-color.html (in German, but you get the idea), you can actually find the non-prefixed scollbar properties document as Microsoft specific CSS extension.

So, this PR renames the scrollbar files to be without the prefix and then adds a range for IE support like this: 

```json
"ie": [
  {
    "version_added": "5"
  },
  {
    "version_added": "8",
    "prefix": "-ms-"
  }
],
```

review? @ExE-Boss

ExE-Boss commented on css/properties/text-align-last.json:
```diff
@@ -64,7 +64,7 @@
             ],
             "ie": {
-              "version_added": true,
+              "version_added": "6",
```
Move this change to #3899.

ExE-Boss commented:
Turns out IE11 still sort&#x2011;of supports this, but I still want the commit history to be unborked, and the above issue addressed.

And this merged using the merge commit strategy (which will require admin merging), like in #1910.

Elchi3 commented:
Okay, so now I have two commits. The first one renames the files. and the second changes the files. I've also removed the text-align-last change, that I added here by mistake. Does this look better now @ExE-Boss? I can admin merge this without the squashing.

ExE-Boss commented:
Well, I guess this is more correct, so this can now be admin merged.

That said, I’d still prefer if the files and path were `-ms-` prefixed, and use `alternative_name` for the unprefixed variant, since these are non-standard Microsoft CSS properties, but I don't know how this repository deals with other non-standard Microsoft CSS properties.

Elchi3 commented:
> That said, I’d still prefer if the files and path were -ms- prefixed, and use alternative_name for the unprefixed variant, since these are non-standard Microsoft CSS properties, but I don't know how this repository deals with other non-standard Microsoft CSS properties.

@ddbeck do you have an opinion on this? I'd be happy to go either way, but I also don't know how we've dealt with this throughout the CSS BCD. The unprefixed variant was around before, so that's why I thought it would be represented as the main identifier and to me it looks like it makes more sense this way. However, I guess, the data model would also allow to swap this around the other way.

vinyldarkscratch commented:
Personally, I think it should be stored in the data without the prefix, with blocks identifying prefixed support, just like how it is.  It feels a little cleaner, and lets us take advantage of the `prefix` property.  Alas, I don’t feel strongly about this; whatever conclusion is made, I’m happy with!

ddbeck commented:
If I've understood correctly, we've got an older, unprefixed property and then a newer, prefixed version of the same property; neither are standard track. I can't recall another case where a non-standard, unprefixed CSS feature was prefixed after the fact. The closest analogy I can think of is Firefox adding `webkit` prefixes, but I think all of those were standard track, where there was a recognizable common name. 

My instinct is that—absent a standard name—we should go with the oldest name in use. In this case, it's the unprefixed name.

Elchi3 commented:
That understanding is mine as well. I also think that the older names are more used as they have been available for much longer. I will admin merge this.

ExE-Boss commented:
Actually, the URLs are still prefixed.

Elchi3 commented:
I will move them.
